src/macros/macrorecorder.h

```diff
@@ -34,18 +25,26 @@ class MacroRecorder : public RecordingManagerBase {
     void stopRecording() override;
     bool isRecordingActive() const override;
 
+    enum Status : uint8_t {
```

Comment:
enum class

Reply:
If this is an enum class, I need to do conversions _every single time_ I use it. It is just a representation of a CO value, so it's not gonna be typesafe ever, so I don't see any advantage for using enum class here.